docs/guides/searching-with-readthedocs.rst

```diff
@@ -0,0 +1,190 @@
+Searching with Read the Docs
+============================
+
+.. meta::
+    :description lang=en:
+        Guide to use the powerful documentation search by Read the Docs to its full extent.
+
+
+This guide is intended to show that how to use the powerful search provided by Read the Docs.
+It is being powered by `Elasticsearch`_.
+You can find more information on the search architecture and how we index document on our
+:doc:`Search <../development/search>` docs.
+
+
+.. contents:: Table of contents
+   :local:
+   :backlinks: none
+   :depth: 3
+
+
+Why "Just" Sphinx Search Is Not Enough?
+---------------------------------------
+
+Sphinx already have built in search functionality,
+but it only supports a basic search across a single documentation.
+While Read the Docs supports a powerful documentation search.
+
+Features of Read the Docs documentation search are:
+
+- Search as you type feature supported.
```

Comment:
I edited the list to match the order of sections.
I don't want to remove the list completely because it also list out the features which are explained below in the doc.